fractal_server/app/schemas/applyworkflow.py

```diff
@@ -53,6 +53,7 @@ class _ApplyWorkflowBase(BaseModel):
     """
 
     worker_init: Optional[str]
+    working_dir: str
```

Comment:
This doesn't work: the `working_dir` attribute is generated server-side, but as of this change it is required that the client knows it in advance before making a POST request.